src/Compilers/CSharp/Portable/Syntax/SyntaxFacts.cs

```diff
@@ -195,6 +195,9 @@ public static bool IsInTypeOnlyContext(ExpressionSyntax node)
 
                     case TypedVariableComponent:
                         return ((TypedVariableComponentSyntax)parent).Type == node;
+
+                    case IncompleteMember:
+                        return ((IncompleteMemberSyntax)parent).Type == node;
```

Comment:
> return ((IncompleteMemberSyntax)parent).Type == node; [](start = 23, length = 54)

Does this change the way SemanticModel APIs behave? Please add compiler tests for them and a direct test for IsInTypeOnlyContext API.  src\Compilers\CSharp\Test\Symbol\Compilation\SemanticModelGetSemanticInfoTests.cs looks like a good place.